### ISSUE TITLE
Track HTTP/2 adoption

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/httparchive.org#43]
Either as % of requests or % of distinct hosts. See https://discuss.httparchive.org/t/http-2-adoption/792/16

**Comment by cqueern** [comment]
Hi @rviscomi, what do you think the level of effort is for this enhancement?

**Comment by rviscomi** [comment]
Hi @cqueern. It will take me some time to get this onto the website but I was able to generate a chart from the raw data.

![image](https://user-images.githubusercontent.com/1120896/42172801-1c44d850-7deb-11e8-97d7-64a06a86b7a3.png)
[Raw data](https://docs.google.com/spreadsheets/d/1vogL0Gb940-MuIkizcfIHjcgfHMKIsgRLVaK6POWOrE/edit?usp=sharing)
[Query](https://bigquery.cloud.google.com/savedquery/226352634162:bcafdea9468342428ef7996f3080e9bd)

**Comment by cqueern** [comment]
Sounds good, thanks @rviscomi !

**Comment by rviscomi** [comment]
Thanks to @malchata this metric is now available at https://httparchive.org/reports/state-of-the-web#h2!

**Comment by rviscomi** [comment]
One minor edge case with this metric.

https://wordpress.httparchive.org/reports/state-of-the-web#h2

When we generate lens-specific results, the JOIN assumes that the `url` field is the URL of the page, not the resource. So the lens results are empty.

This can be solved by replacing the table wildcard with a subquery that aliases `page` as `url`. Eg replace this 

```sql
`httparchive.requests.*`
```

with this

```sql
(SELECT page AS url, payload, _TABLE_SUFFIX AS _TABLE_SUFFIX FROM `httparchive.requests.*`)
```

**Comment by malchata** [comment]
This seems like it should be opened for the bigquery project, as we're modifying the SQL brought in from there. Am I wrong, or does that seem accurate?

Either way, happy to open a PR for this in the bigquery project, unless I missed something here.

**Comment by rviscomi** [comment]
Yeah it has more to do with the SQL. It's ok to reuse this issue as it's a visible problem in the UI. 

FWIW we have the same issue with the HTTPS audit. I'm currently looking into it.
